api/src/main/java/com/cloud/event/EventTypes.java

```diff
@@ -266,6 +267,7 @@
     public static final String EVENT_SSVM_STOP = "SSVM.STOP";
     public static final String EVENT_SSVM_REBOOT = "SSVM.REBOOT";
     public static final String EVENT_SSVM_HA = "SSVM.HA";
+    public static final String EVENT_SSVM_DIAGNOSTICS = "SSVM.DIAGNOSTICS";
```

Comment:
EVENT_SSVM_DIAGNOSTICS is not used outside this file. Do we really need it?

Reply:
Removed EVENT_SSVM_DIAGNOSTICS